### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by TimB-87**

### CODE REVIEW

File: src/main/java/frc/robot/maps/FrostBiteMap.java [hunk 1]
```diff
@@ -103,7 +106,7 @@ encoderRR, rearRightSteer, new CSSparkMax(1,
 
         final double maxRotationRadianPerSecond = Math.PI;
 
-        final DrivePID pid = new DrivePID(0.8, 0, 0, 0.01, 0, 0.001);
```
Comment (bot190) [inline src/main/java/frc/robot/maps/FrostBiteMap.java]: This isn't the PID value we want to be tuning.

File: src/main/java/frc/robot/maps/FrostBiteMap.java [hunk 2]
```diff
@@ -66,7 +69,7 @@ public SwerveDriveMap getDriveMap() {
         // All Distances are in Meters
         // Front Left Module
         final CANCoder encoderFL = new CANCoder(4);
-        encoderFL.configMagnetOffset(-29.8828125);
+        encoderFL.configMagnetOffset(-19.8828125);
```
Comment (bot190) [inline src/main/java/frc/robot/maps/FrostBiteMap.java:70]: Lets be methodical about tuning these. I would be very surprised if this is off by 10 degrees.

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -48,34 +48,44 @@ public class Robot extends CommandRobot {
...
                 return gamePieceSub.get() == "Cone";
             }));
 
+    public CommandBase scoreMidNode = sequence(
+            armRotate.moveTo(EnumLevel.MEDIUM_SCORE), (arm.moveTo(EnumLevel.MEDIUM_SCORE)),
+            (armRotate.moveTo(EnumLevel.MEDIUM_SCORE_ACTUAL)));
+
+    public CommandBase scoreHighNode = sequence(
+            armRotate.moveTo(EnumLevel.HIGH_SCORE), (arm.moveTo(EnumLevel.HIGH_SCORE)),
+            (armRotate.moveTo(EnumLevel.HIGH_SCORE_ACTUAL)));
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: Shouldn't this sequence also retract the arm?

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -127,13 +140,18 @@ public void configureButtonBindings() {
         copilotController.leftBumper().onTrue(grabCone());
         // will need buttons for the enums
         copilotController.y().whileTrue(armRotate.moveTo(EnumLevel.HPS_PICKUP));
+        // copilotController.povUp()
+        // .whileTrue(scoreHighNode);
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: isn't this the same as the line below? Can we remove these?

**Review by bot190 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (TimB-87) [reply src/main/java/frc/robot/Robot.java]: Yes it most certainly should. I thought that was in there but apparently not, I'll put it in.

**Pull request merged**